client/src/components/Post/PostComments/Comment/Comment.jsx

```diff
@@ -1,26 +1,52 @@
-import React, { Fragment } from 'react'
+import React, { Fragment, useEffect, useState } from 'react'
+import { connect } from 'react-redux'
 import useStyles from './commentStyles'
 import PropTypes from 'prop-types'
 
+import IconButton from '@material-ui/core/IconButton'
+import DeleteIcon from '@material-ui/icons/Delete'
+
+import { deleteComment } from '../../../../actions/post'
 import getDate from '../../../../utils/date/getDate'
 
-const Comment = ({ comment }) => {
+
+const Comment = ( { postId, comment, user, deleteComment } )  => {
   const classes = useStyles()
 
-  const { author, message, date } = comment
+  const [showDeleteBtn, setShowDeleteBtn] = useState(false)
+  const { id, author, message, date } = comment
+
+  useEffect(
+    () => setShowDeleteBtn(author.username === user.username ),
+    [author.username, user.username]
+  )
 
   return (
     <Fragment>
```

Comment:
There is no need in `<Fragment>` as far as whole component is wrapped in div element